distribution/resources/cli-conf/toolkit-config.toml

```diff
@@ -3,7 +3,7 @@ httpRequestTimeout = 1000000
 
 [token]
 baseURL = ""
-restVersion = "v1.0"
```

Comment:
Why do we need 0.14 which is AM 260 rest version? Even if we need to use compatibility version then this has to be 0.15 IMO

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/966/commits/fe1f68c2c9d1d39e961d2664256249498d69c5aa